### ISSUE TITLE
Details about current situation ?

### ISSUE THREAD

**Issue by ghost** [gitabcworld/FewShotLearning#2]
Did you solve the problem about the gradients not flowing to LSTM modules?

I was trying to implement Learning to Learn by gradient decent by gradient decent using tensorflow, and I encounter a similar problem, but I solved the problem by updateing cross-entropy loss in every step using newest weights of classifier(optimizee) , could this be a potential problem?

**Comment by gitabcworld** [comment]
Thanks for your suggestion,
I haven't been able work on this project due I have switched to another project. I will definitely come back to this project because I need to solve it, but due to my schedule I won't be able until begining of December. I will try your solution and see if it works. 
Thanks again.

**Comment by dragen1860** [comment]
@gitabcworld pls get some time to solve your great code. We need it. Thanks.
